life_server/Functions/Systems/fn_vehicleCreate.sqf

```diff
@@ -78,7 +63,21 @@ _vehicle setVariable ["trunk_in_use",false,true];
 _vehicle setVariable ["vehicle_info_owners",[[_uid,name _unit]],true];
 _vehicle setPlateNumber _plate;
 _vehicle setVariable ["plate", _plate, true]; //'Air' don't work properly for setPlateNumber
+
+private _vid = -1;
 if (_purchase) then {
+    private _type = call {

```

Comment:
_type could be nil; either check for that or use a default case (doesn't really makes sense tho I guess)

Reply:
Got 2 options here, I change "Car" to "LandVehicle" and that'll cover more off, or use "Air", "Ship" and then default is "Car"

Reply:
probably would use option 2, is also more versatile